pkg/volume/iscsi/iscsi_util.go

```diff
@@ -98,7 +98,7 @@ func updateISCSIDiscoverydb(b iscsiDiskMounter, tp string) error {
 			// explicitly not using execWithLog so secrets are not logged
 			out, err := b.exec.Run("iscsiadm", "-m", "discoverydb", "-t", "sendtargets", "-p", tp, "-I", b.Iface, "-o", "update", "-n", k, "-v", v)
 			if err != nil {
-				return fmt.Errorf("iscsi: failed to update discoverydb key %q error: %v", k, string(out))
+				return fmt.Errorf("iscsi: failed to update discoverydb key %q error: %v", k, out)
```

Comment:
`b.exec.Run` returns `byte[]` and this will print array of bytes instead of a string.